### PR TITLE
Fix two bugs that prevent post-commands running in exception case.

### PR DESCRIPTION
**Pull request by vext01**
OK?

### CODE REVIEW

File: krun/scheduler.py [hunk 1]
```diff
@@ -511,6 +511,7 @@ def run(self):
             # exception occurred in the above try block, there's a chance that
             # they have not have been loaded.
             if results is None:
+	        Results.ok_to_instantiate = True
```
Comment (snim2) [inline krun/scheduler.py]: The indentation looks wrong here, is that just GitHub?

File: krun/scheduler.py [hunk 2]
```diff
@@ -507,6 +511,7 @@ def run(self):
...
```
Comment (vext01) [inline krun/scheduler.py]: funny indent?

File: krun/scheduler.py (hunk 1 above)
Comment (vext01) [inline krun/scheduler.py]: fixed.

**Comment by snim2** [comment]
OK, if the tests are good, let's squash

**Comment by vext01** [comment]
splat.

**Pull request merged**